microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -86,16 +85,84 @@ The capability to back up and restore the {product-title} database is now availa
 //[id="microshift-4-14-storage"]
 //=== Storage
 
+[id="microshift-4-14-running-apps"]
+=== Running Applications
+//still need to add
+
+[id="microshift-backup-and-restore"]
+=== Backup and restore
+The capability to back up and restore the {product-title} database is now available. You can manually back up and restore data on all supported systems at any time. See the xref:../microshift_backup_and_restore/microshift-backup-and-restore.adoc#microshift-backup-and-restore[Backing up and restoring {product-title} data] documentation for more information.
+
 //[id="microshift-4-14-lvms-system-requirements"]
 //==== LVMS system requirements
 
-//[id="microshift-4-14-deprecated-removed-features"]
-//== Deprecated and removed features
+[id="microshift-4-14-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+// Note: use [discrete] for these sub-headings.
+
+[discrete]
+[id="microshift-4-14-networking-changes"]
+=== Networking changes
 
-//[id="microshift-4-14-bug-fixes"]
-//== Bug fixes
+Networking updates to {product-title} {product-version} include traffic flow pattern, gateway, and custom configuration changes.
 
-//etc
+[discrete]
+[id="microshift-4-14-traffic-flow-change"]
+==== North-south traffic flow changed
+
+The external gateway bridge and the physical device on the host are no longer connected. The north-south traffic between the network service and the OVN external switch flows from the host kernel to {product-title} through the external gateway bridge. See the xref:../microshift_networking/microshift-networking.adoc#microshift-description-connections-network-topology_microshift-networking[MicroShift networking] documentation for more information.
+
+[id="microshift-4-14-deprecated-and-removed"]
+== Deprecated and removed features
```

Comment:
@DanielFroehlich Are we following a Deprecation schedule or system with MicroShift's lifecycle? We listed the removed network flags as "deprecated," but I think they are actually removed. Could you clarify if we actually have Deprecated features for 4.14? 
"Deprecated functionality is still included in {product-title} and continues to be supported;"